### PR TITLE
fix(formatting): fix remove code formatting - fixes: #40

### PR DESCRIPTION
**Pull request by emahuni**
See issue #40 for more details...

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 module.exports = {
```
Comment (derrickmehaffy) [inline .prettierrc.js:1]: Can we keep this file please as it's for this repo, not projects being migrated.

File: package.json [hunk 2]
```diff
@@ -32,8 +32,7 @@
     "inquirer-fuzzy-path": "^2.3.0",
     "jscodeshift": "^0.13.0",
     "lodash": "^4.17.21",
-    "pluralize": "^8.0.0",
-    "prettier": "^2.4.1"
```
Comment (derrickmehaffy) [inline package.json:36]: Same here can we keep prettier for this repo

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

File: .prettierrc.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
   endOfLine: 'lf',
-  semi: true,
-  singleQuote: true,
+  semi: false,
+  singleQuote: false,
+  quoteProps: 'preserve',
   tabWidth: 2,
   trailingComma: 'es5',
-  printWidth: 100,
+  printWidth: 120,
```
Comment (markkaylor) [inline .prettierrc.js:8]: Please don't change these values they are set to match the config we use in the Strapi core repo

**Comment by derrickmehaffy** [comment]
Since there has been no update to this PR I'm going to mark it as closed.

**Pull request closed without merging**